.env

```diff
@@ -0,0 +1,9 @@
+MYSQL_USERNAME=root
```

Comment:
Remove .env file
git rm --cached .env 

add .env.template instead with actual values removed - otherwise we would need to search for all env var throughout the project to start it

Reply:
env file still there